### PR TITLE
Run on AWS instance option fix

### LINKED ISSUE #865: AWS run option.
**Is your feature request related to a problem? Please describe.**
AWS run option needs to be re-introduced and tested with the new run page UI.

**Describe the solution you'd like**
AWS run option is also present on "Run monkey" page when running on AWS.
We also need to change the Docs.

**Describe alternatives you've considered**
Leave AWS run option disabled.

### PR DESCRIPTION
**Pull request by VakarisZ**
# What does this PR do? 

Fixes #865 

image

## PR Checklist
* [ ] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing? 

## Testing Checklist

* [ ] Added relevant unit tests?
* [x] Have you successfully tested your changes locally? Elaborate:
    > Tested by running monkeys on AWS via this fixed feature
* [ ] If applicable, add screenshots or log transcripts of the feature working

## Explain Changes

In addition moved run page files around, fixed eslint not to warn about backtick usage.

### CODE REVIEW

File: monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/RunOnAWS/AWSInstanceTable.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+const pageSize = 10;
+
+function AWSInstanceTable(props) {
+
+  const [allToggled, setAllToggled] = useState(false);
+  let checkboxTable = null;
+
+  function toggleSelection(key) {
+    key = key.replace('select-', '');
```
Comment (VakarisZ) [inline monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/RunOnAWS/AWSInstanceTable.js:29]: Maybe that's why it wasn't working before. AWS changed the format of instance ID's returned and it broke? This line compensates the change

File: monkey/monkey_island/cc/ui/src/components/ui-components/LoadingIcon.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import {faSync} from '@fortawesome/free-solid-svg-icons';
+import React from 'react';
```
Comment (acepace) [inline monkey/monkey_island/cc/ui/src/components/ui-components/LoadingIcon.js:3]: Nit: Do we need all of react imported there?

**Comment by acepace** [comment]
@VakarisZ this looks good, how did you verify it works? Because not sure I can easily test

**Comment by VakarisZ** [comment]
I ran it on AWS

**Comment by acepace** [comment]
Then no comment preventing. Anyone?

**Pull request merged**